.github/workflows/java-eks-e2e-test.yml

```diff
@@ -184,6 +184,12 @@ jobs:
           RDS_MYSQL_CLUSTER_CREDENTIAL_SECRET_NAME=$(aws secretsmanager list-secrets --region ${{inputs.aws-region}} --query "SecretList[?Tags[?Value=='arn:aws:rds:${{inputs.aws-region}}:${{env.ACCOUNT_ID}}:cluster:$RDS_MYSQL_CLUSTER_IDENTIFIER']].Name" --output text)
           echo RDS_MYSQL_CLUSTER_CREDENTIAL_SECRET_NAME="$RDS_MYSQL_CLUSTER_CREDENTIAL_SECRET_NAME" >> $GITHUB_ENV
 
+      - name: Get RDS database instance metadata
+        run: |
+          INSTANCE_ID=$(aws rds describe-db-instances --query "DBInstances[0].DBInstanceIdentifier" --output text)
+          REMOTE_DB_USER=$(aws rds describe-db-instances --db-instance-identifier $INSTANCE_ID --query "DBInstances[0].MasterUsername" --output text)
+          echo REMOTE_DB_USER="$REMOTE_DB_USER" >> $GITHUB_ENV
+
```

Comment:
Can we merge this with the above step? And if it's reusable between Java and Python, we can look to make it a reusable action instead of a piece of duplicated code. We would put this in `.github/workflows/actions`

Similar to: https://github.com/aws-observability/aws-application-signals-test-framework/blob/main/.github/workflows/actions/patch_image_and_check_diff/action.yml

Reply:
Sure, will update to the most efficient way.